com.unity.netcode.gameobjects/Tests/Runtime/Metrics/ServerLogsMetricTests.cs

```diff
@@ -64,7 +63,6 @@ public IEnumerator TrackServerLogReceivedMetric()
             var receivedMetric = receivedMetrics.First();
             Assert.AreEqual(Client.LocalClientId, receivedMetric.Connection.Id);
             Assert.AreEqual((uint)NetworkLog.LogType.Warning, (uint)receivedMetric.LogLevel);
-            Assert.AreEqual(message.Length + k_ServerLogReceivedMessageOverhead, receivedMetric.BytesCount);
```

Comment:
So this isn't a fix. I understand it might be harder to get the size now, but we are reporting that size and we need to validate the size is accurate. Fixing this test should not be removing a line that is failing. I cannot in good conscious approve this PR without these tests without the size.